lms/djangoapps/courseware/tabs.py

```diff
@@ -358,8 +338,7 @@ def get_course_tab_list(user, course):
             if tab.type != 'courseware':
                 continue
             tab.name = _("Entrance Exam")
-        # TODO: LEARNER-611 - once the course_info tab is removed, remove this code
-        if UNIFIED_COURSE_TAB_FLAG.is_enabled(course.id) and tab.type == 'course_info':
+        if tab.type == 'course_info':
```

Comment:
to clarify, the tabe we want to have remain is the course home page, not the info page. All references to courseware I expect would go away with this work, (where "courseware / info" were named of the previous two tabs in the code but also the name of the tab in the UI.

Reply:
@marcotuts 

I may have misunderstood some things for this then. Can you let me know, for each of the following, what it is used for and whether it should remain or be taken out?

* `course_info`
* `courseware`

Are there any other references/strings that are used that should be removed, and if so, can you link me an example of it in the current master edx-platform code?

Reply:
@pcockwell Heh, that's kind of the catch-22 here... if we knew every string and reference that should and shouldn't be removed ahead of time, we'd have already been done by now ;)

Given the ambiguity of requirements here, plus the size of this, we'd like to punt on this for now and revisit this at a later date, when a TNL engineer can devote more attention.